File: tests/patterns/TEMPLATE/beta-c-like/metavar_class_def.java
// ERROR:
public class Foo {
    int foo = 5;

    public static void foo() {
        int foo1 = 4;
    }
}

// ERROR:
private class Bar extends Foo {
    int bar = 3;
}

// ERROR:
abstract class Foobar {
    public abstract void foobar();
}

File: semgrep-core/tests/kotlin/metavar_class_def.kt
// ERROR:
public class Foo {
    val foo: Int = 5

    public fun foo(): Void {
        val foo1: Int = 4
    }
}

// ERROR:
private class Bar: Foo {
    val bar: Int = 3
}

// ERROR:
abstract class Foobar {
    public abstract fun foobar(): Void
}
